src/maintenancePage/maintenancePage.component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { createMount } from '@material-ui/core/test-utils';
+import {
+  CombinedMaintenancePageProps,
+  MaintenancePageWithStyles,
```

Comment:
By importing this version rather than the connected default export we don't test mapping the store to the components props. It would be better (in terms of coverage) to import `default as MaintenancePage` and then create a `testStore` for the test rather than directly passing in `message` as a prop.

Reply:
That makes sense, thanks!